orcoursetrion/actions/github.py

```diff
@@ -93,6 +94,35 @@ def rerun_studio(course, term, new_term, description=None):
     return repo
 
 
+def release_studio(course, term):
+    """Moves a studio course to be ready for production.
+
+    Currently this will just add a hook to the production server, but
+    it will eventually take care of everything else needed for a
+    transfer as well.
```

Comment:
What is the extent of the other things that will be needed here for a full transfer again? Is it just setting the course team and perhaps triggering the webhook?